application/helpers/general_functions_helper.php

```diff
@@ -137,6 +137,13 @@ function full_languages_dropdown($id='', $selected_id=''  )
         $html .= '<option value="" '. $selected .' > ------ </option>';
     }
 
+    if ($extra_search_options)
+    {
+        $selected = ($selected_id == '-1') ? ' selected ': '';
```

Comment:
So -1 is a magic value which, instead of being a language ID, means "all except english"? Could we then declare it as a constant, and reference that constant here instead of the literal -1?